src/test/kotlin/racing/model/RacingGameTest.kt

```diff
@@ -2,34 +2,85 @@ package racing.model
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import racing.util.NumberGenerator
 
 internal class RacingGameTest {
 
-    private val racingGame = RacingGame(listOf(Car("woni"), Car("pobi"), Car("jason")))
+    @Test
+    fun `moveCars를 통해 woni를 이동 시키면 한번 움직인 결과로 woni의 postion은 1`() {
+        //given
+        val cars = listOf(Car("woni"), Car("pobi"), Car("jason"))
+
+        //when
+        val movedCars: List<Car> = RacingGame.moveCars(cars, OneMoveRandomNumbers())
+
```

Comment:
NumberGenerator를 fun interface로 선언하면, 해당 인터페이스를 마지막 파라미터로 넘길 때 람다로 넘길 수 있게 됩니다.
즉, inner class 등으로 구현체를 직접 만들지 않아도 됩니다 :)
SAM에 대해 공부해보시길 추천드릴게요 😊
```suggestion

        //when
        val movedCars: List<Car> = RacingGame.moveCars(cars) { 1 }

```